### PR TITLE
Assignment 4:Task 6 y 7

### PR DESCRIPTION
**Pull request by NatalyMMR**

### CODE REVIEW

File: Assignment4/NatalyMMR-e22a304/task06.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+u = Namespace("http://somewhere#")
+g.add((u.University, RDF.type, RDFS.Class))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
+
+#TO DO
+r = Namespace("http://somewhere#")
```
Comment (dgarijo) [inline Assignment4/NatalyMMR-e22a304/task06.py:46]: why a duplicated namespace (you had ns above)?

File: Assignment4/NatalyMMR-e22a304/task06.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+#TO DO
+r = Namespace("http://somewhere#")
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+
+g.add((r.UPM, RDF.type, r.University))
+g.add((r.JohnSmith, VCARD.Work, r.UPM))
```
Comment (dgarijo) [inline Assignment4/NatalyMMR-e22a304/task06.py:83]: Work does not exist in vcard. Create a new property in your ns

**Review by dgarijo — CHANGES_REQUESTED** [review]
Small errors in task 6 need to be reviewed

File: Assignment4/NatalyMMR-e22a304/task06.py (hunk 2 above)
Comment (NatalyMMR) [reply Assignment4/NatalyMMR-e22a304/task06.py]: Done!

File: Assignment4/NatalyMMR-e22a304/task07.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+"""Task07.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1Z0k15vN8rPopqqRfsjxeU_qTzsuMflW5
+
+**Task 07: Querying RDF(s)**"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/NatalyMMR-e22a304/task07.py:11]: same comment about python and notebook magic

**Review by dgarijo — APPROVED** [review]

**Pull request merged**